Content.Client/GameTicking/ClientGameTicker.cs

```diff
@@ -40,12 +44,14 @@ public void Initialize()
             _netManager.RegisterNetMessage<MsgTickerLobbyStatus>(nameof(MsgTickerLobbyStatus), LobbyStatus);
             _netManager.RegisterNetMessage<MsgTickerLobbyInfo>(nameof(MsgTickerLobbyInfo), LobbyInfo);
             _netManager.RegisterNetMessage<MsgTickerLobbyCountdown>(nameof(MsgTickerLobbyCountdown), LobbyCountdown);
+            _netManager.RegisterNetMessage<MsgTickerLobbyReady>(nameof(MsgTickerLobbyReady), LobbyReady);
             _netManager.RegisterNetMessage<MsgRoundEndMessage>(nameof(MsgRoundEndMessage), RoundEnd);
             _netManager.RegisterNetMessage<MsgRequestWindowAttention>(nameof(MsgRequestWindowAttention), msg =>
             {
                 IoCManager.Resolve<IClyde>().RequestWindowAttention();
             });
 
+            Ready = new Dictionary<NetSessionId, bool>();
```

Comment:
This dictionary doesn't has a key removed from it in the event that a player disconnects. Since the ready dictionary is merged from the server when the NetMessage arrives, this means that it will never lose outdated entries.

Reply:
True. I will remove it on connection changed